infer/lib/python/inferlib/jwlib.py

```diff
@@ -24,14 +24,46 @@ def __init__(self, javac_args):
     def start(self):
         infer = os.path.join(config.BIN_DIRECTORY, 'infer')
         # pass --continue to prevent removing the results-dir
+        new_javac_args = []
+        list_count = 0
+        file_count = 0
+        source_list = "infer_sourcefiles"
+        if os.path.exists(source_list+str(list_count)):
```

Comment:
Please use `tempfile.NamedTemporaryFile()` instead, similar to [here](https://github.com/facebook/infer/blob/690bae30d0b6163f23075bf22fa5b4faf248104f/infer/lib/python/inferlib/capture/gradle.py#L196-L202) for instance.

Reply:
ok